packages/orbit-components/src/Truncate/index.tsx

```diff
@@ -1,32 +1,29 @@
 "use client";
 
 import * as React from "react";
-import styled from "styled-components";
+import cx from "clsx";
 
 import type { Props } from "./types";
 
-const StyledTruncate = styled.div<{ maxWidth?: string }>`
-  ${({ maxWidth }) => `
-    min-width: 0;
-    flex: 0 1 ${maxWidth === "none" ? "100%" : maxWidth};
-    max-width: ${maxWidth};
-  `};
-`;
-
-const StyledTruncateContent = styled.div`
-  &,
-  .orbit-text,
-  .orbit-heading {
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-  }
-`;
-
 const Truncate = ({ children, maxWidth = "100%", dataTest }: Props) => (
-  <StyledTruncate maxWidth={maxWidth} data-test={dataTest}>
-    <StyledTruncateContent>{children}</StyledTruncateContent>
-  </StyledTruncate>
+  <div
+    data-test={dataTest}
+    className={cx(
+      "min-w-0 max-w-[--max-width] shrink grow-0",
+      maxWidth === "none" ? "basis-full" : "basis-[--max-width]",
+    )}
+    style={{ "--max-width": maxWidth } as React.CSSProperties}
+  >
+    <div
+      className={cx(
+        "overflow-hidden text-ellipsis whitespace-nowrap",
```

Comment:
Tailwind actually has a nice and official shorthand for these three classes: the class `truncate`. So we can use it here. Just make sure it also works with custom selectors, but I don't see why it shouldn't…

```suggestion
        "truncate",
```

It's interesting because I remember that in the past the linter actually suggested me (via warning) that these three classes could be replaced by the `truncate` class (that's actually how I first learned about it) but now I see it is also not giving me any warning either 🤔

Reply:
TIL, thanks!